R/derive_param_tte.R

```diff
@@ -738,14 +738,7 @@ extend_source_datasets <- function(source_datasets,
   by_groups <- unique(bind_rows(by_groups))
   for (i in seq_along(source_datasets)) {
     if (extend[[i]]) {
-      source_datasets[[i]] <-
-        full_join(
-          mutate(by_groups, temp_dummy = 1),
-          mutate(source_datasets[[i]], temp_dummy = 1),
-          by = "temp_dummy",
-          relationship = "many-to-many"
```

Comment:
does this impact the `admiral_environment` stuff??

Reply:
From my understanding it shoudn't, only objects in there right now are "many-to-one" , "one-to-many" , the subject keys stuff, and duplicates

Reply:
> does this impact the `admiral_environment` stuff??

No, this is something completely different.